htdocs/expedition/class/expedition.class.php

```diff
@@ -422,7 +429,7 @@ function fetch($id, $ref='', $ref_ext='', $ref_int='')
         if ($ref_ext) $sql.= " AND e.ref_ext='".$this->db->escape($ref_ext)."'";
         if ($ref_int) $sql.= " AND e.ref_int='".$this->db->escape($ref_int)."'";
 
-		dol_syslog(get_class($this)."::fetch", LOG_DEBUG);
```

Comment:
Woops! Here you removed LOG_DEBUG

And BTW I'll try to take a look at the wiki so that this things get documented :smile: